src/js-form-validator.ts

```diff
@@ -95,6 +100,9 @@ export class FormValidator {
 
       if (valid === 0)
         tooltip = invalidMsg;
+      else if (valid === -2) {
```

Comment:
Could we be checking against an enum in order for this to be easier to read?

Reply:
@samiam236 that is a great idea. It had been requested on a previous request that would include a breaking change in versions from my previous research. It was stated on this PR in a conversation with Nick.

https://github.com/GTMSportswear/js-form-validator/pull/2

Reply:
@someyoungideas So are you saying it would be a breaking change to check against an enum?

Reply:
@samiam236 I started down the path again to figure out how I came to he conclusion last time. After starting things off what comes to mind is that we currently have a `data-empty_allowed` attribute that gets set to `1` if you want it applied. Unfortunately the value we set to `valid` variable when the value is empty is `-1` which seemed confusing and counter intuitive. However, I went ahead and implemented the enum, and left single check that that attribute is equal to `1`, leaving that decision for later.